### PR TITLE
add dedicated grpc api widget

### PR DESCRIPTION
**Pull request by kissmikijr**
Signed-off-by: Kiss Miklos <miklos@roadie.io>

## Hey, I just made a Pull Request!

The official docs states the support for `grpc` in the api view. However there are no dedicated component and it uses the `<PlainApiDefinitionWidget />`, this widget is a wrapper around the `<CodeSnippet />` componenet which uses directly the `<LigthAsync />` component from the `react-syntac-highlighter` package. 
The issue is that the api definitions type is directly passed into the react-syntax-highlighter and the package does not contain syntax highlighting for a language called `grpc` because actually a grpc definitions are the `.proto` files. So we'd need to use the language `protobuf` to get syntax highlighting. 
I created a dedicated widget for the grpc api definitions just to map the `grpc` type to `protobuf` so we could get syntax highlighting for the definitions without the need to change the type from `grpc` to `protobuf`
     
Before: 
image
After:
image

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [x] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/api-docs/src/components/GrpcApiDefinitionWidget/GrpcApiDefinitionWidget.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { CodeSnippet } from '@backstage/core-components';
+import { useTheme } from '@material-ui/core/styles';
+import { BackstageTheme } from '@backstage/theme';
+
+export type GrpcApiDefinitionWidgetProps = {
+  definition: any;
```
Comment (jhaals) [inline plugins/api-docs/src/components/GrpcApiDefinitionWidget/GrpcApiDefinitionWidget.tsx:23]: Type string should work here right?

**Review by jhaals — COMMENTED** [review]
Nice! This looks 100x nicer 👍

File: plugins/api-docs/src/components/GrpcApiDefinitionWidget/GrpcApiDefinitionWidget.tsx (hunk 1 above)
Comment (kissmikijr) [reply plugins/api-docs/src/components/GrpcApiDefinitionWidget/GrpcApiDefinitionWidget.tsx:23]: Yes!

**Review by freben — APPROVED** [review]
<3

**Pull request merged**